docs/user-guide/scaling.rst

```diff
@@ -31,8 +31,9 @@ Pulp consists of several components:
   Additionally, it performs task cancellations in the event of a worker
   shutdown or failure. The celerybeat process also initiates scheduled tasks,
   and automatically cancels tasks that have failed more than *X* times. This
-  process also initiates periodic jobs that Pulp runs internally. In a Pulp
-  cluster, exactly one of these should be running!
+  process also initiates periodic jobs that Pulp runs internally. Multiple
+  instances of the ``pulp_celerybeat`` process can be started together to ensure
```

Comment:
s/started together/run at the same time/

It's not important when they start, but that they can run together.